### ISSUE TITLE
Problem: nothing provides dbus needed by headset-1.6.4-1.x86_64 on openSUSE

### ISSUE THREAD

**Issue by karlos1841** [headsetapp/headset-electron#112]
### Description
During package installation on a full-featured desktop environment, the following error occurs:

 Problem: nothing provides dbus needed by headset-1.6.4-1.x86_64
 Solution 1: do not install headset-1.6.4-1.x86_64
 Solution 2: break headset-1.6.4-1.x86_64 by ignoring some of its dependencies

### Expected behaviour
The package should be installed without issues.
On openSUSE it is called "dbus-1". Maybe it should check if service is running?

### Your Environment
OS: openSUSE Leap 42.3
Headset version: headset-1.6.4-1.x86_64.rpm

**Comment by fcastilloec** [comment]
Thanks for letting me know that. We don't test on openSUSE and that's why we didn't found this bug earlier.
It's a simple fix, let me try something and I'll send you an installer to try.

**Comment by fcastilloec** [comment]
@karlos1841 can you test if this package installs on your system? [headset-1.6.4.x86_64.rpm](https://drive.google.com/open?id=1PTmN6KC_8OFJPBTzyQcjifQAOGKRVauK)
If it works, feel free to keep using it, since it would have added the repo automatically and you'll receive the new version with the same fix.

**Comment by fcastilloec** [comment]
So, it seems that we won't be able to support OpenSUSE for the moment. I'm posting a new link [here](https://www.dropbox.com/s/2b8p97kkjdyqr42/headset-1.6.4.x86_64.rpm?dl=0) that will work on openSUSE but we won't be able to fix this in the near future. We need to manually rebuild just for openSUSE for every release.
The problem is that all `rpm` based distros use the same name for `dbus` except for openSUSE. The latest version of `rpmbuild` includes a way to use boolean operators, i.e. `(dbus or dbus-1)` but this version is not included on Travis-CI (based on Ubuntu); in fact, it's not included in any version of Ubuntu except for the upcoming 18.04.
We could try to install dependencies through `rpm` scripts, but this is not supported (at the moment) by `electron-installer-redhat` which we use to build our packages.

Long story short, feel free to use the provided installer through the link, but don't expect future versions or updates for it. At least not in the near future.
